guides/source/caching_with_rails.md

```diff
@@ -241,6 +332,19 @@ There are some common options used by all cache implementations. These can be pa
 
 * `:race_condition_ttl` - This option is used in conjunction with the `:expires_in` option. It will prevent race conditions when cache entries expire by preventing multiple processes from simultaneously regenerating the same entry (also known as the dog pile effect). This option sets the number of seconds that an expired entry can be reused while a new value is being regenerated. It's a good practice to set this value if you use the `:expires_in` option.
 
+#### Custom Cache Stores
+
+You can create your own custom cache store by simply extending
+`ActiveSupport::Cache::Store` and implementing the appropriate methods. This way,
```

Comment:
As reader I'm wondering "what are the appropriate methods?" And since I can't see them here, I wonder where I can find out what they are. Lets help the readers out and not cause a mental drip :smile: